### PR TITLE
update README for beta release

### PR DESCRIPTION
**Pull request by prayansh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,344 +1,57 @@
 # Analytics-Swift
 
-NOTE: This project is currently in the Pilot phase and is covered by Segment's [First Access & Beta Preview Terms](https://segment.com/legal/first-access-beta-preview/).  We encourage you
-to try out this new library. Please provide feedback via Github issues/PRs, and feel free to submit pull requests.  This library will eventually 
-supplant our `analytics-ios` library, but customers should not use this library for production applications during our Pilot phase. 
+NOTE: This project is currently in the Beta phase and is covered by Segment's [First Access & Beta Preview Terms](https://segment.com/legal/first-access-beta-preview/).  We encourage you
+to try out this new library. Please provide feedback via Github issues/PRs, and feel free to submit pull requests.
 
-The hassle-free way to add Segment analytics to your Swift app (iOS/tvOS/watchOS/macOS/Linux).
+The hassle-free way to add Segment analytics to your Swift app (iOS/tvOS/watchOS/macOS/Linux). Analytics helps you measure your users, product, and business. It unlocks insights into your app's funnel, core business metrics, and whether you have product-market fit.
```
Comment (migs647) [inline README.md]: If we're going to list all of the platforms, we should also include iPadOS.

Comment (prayansh) [reply README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -1,344 +1,57 @@
...
 
-## Usage
-### Setting up the client
-The Analytics client will typically be set up at application launch time, such as `applicationDidFinishLaunching`.
+4. **Query your data in SQL**. Slice, dice, and analyze your data in detail with Segment SQL. We'll transform and load your customer behavioral data directly from your apps into Amazon Redshift, Google BigQuery, or Postgres. Save weeks of engineering time by not having to invent your own data warehouse and ETL pipeline.
 
-Typically the following call may be all that's required.
+   For example, you can capture data on any app:
+    ```kotlin
+    analytics.track('Order Completed', Properties(price = 99.84))
```
Comment (migs647) [inline README.md]: This line is incorrect. Not sure where `Properties` is coming from so hard to discuss how to fix.

**Review by bsneed — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bsneed) [reply README.md]: It's also mentioning Kotlin in the Swift repo.  Examples of this can be found in Stacy's docs.

**Pull request merged**